### DISCUSSION TITLE
how can i disable air from turning pink

### DISCUSSION THREAD

**Discussion by nitram-gaming** [maruohon/litematica#924] [General]
![2024-12-16_18 32 09](https://github.com/user-attachments/assets/d1fbbe5b-f348-4903-9f1e-cdad237e1e8e)
how can I disable the air turning pink 
issue started showing up and have no idea how to fix this I only have this issue on my mainworld (+backups of that world)
(I think) I have narrowed it down to the level.dat file (not sure about this)

**Comment by maruohon** [comment]
I don't see air as pink/extra blocks in this screenshot? There is some grass thingy and some vines etc. and then you have some pink blocks in the build?

Maybe there is also one pink block at the player's head? Can you do a third person screenshot in F5 or free camera mode? In that case it's probably some mod putting something like a light block at your head when you wear enchanted armor or whatever the feature might be. (This is a semi-common issue I see reported.)

**Reply by nitram-gaming** [reply]
you were right, dynamic light was rendering a transparent block over my head which caused it
thank you
